### PR TITLE
Resolve #5. Add GetBucket functionality

### LINKED ISSUE #5: Add GetBucket code
create a Callable Object called GetBucket
create sibling function called get_bucket

init args will be _client = Client
call args will be bucket_name
It must initialise client and call its client.get_bucket(bucket_name)

maybe it can have aditional optional arg: projection: str = 'noAcl' and or fields if that exists

### PR DESCRIPTION
**Pull request by ChameleonTartu**

### CODE REVIEW

File: storage_bucket/get.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+from typing_extensions import final
+
+
+@final
+@dataclass(frozen=True, slots=True)
+class GetBucket(object):
+    """Get a GCP storage bucket."""
+
+    _client = Client
+    _bucket = Bucket
```
Comment (thomasborgen) [inline storage_bucket/get.py]: this can be removed

File: tests/get_test.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+from storage_bucket.get import GetBucket, get_bucket
+
+
+class TestGetBucket(object):
+    """Test Get Bucket."""
+
+    def test_get_bucket_success(self):
+        """Create bucket, get Success Modal."""
+        bucket_result = GetBucket()(
+            storage_bucket_name='test-bucket-my-name-is',
```
Comment (thomasborgen) [inline tests/get_test.py]: Please refer to tests/upload_test.py
add: 
STORAGE_BUCKET_NAME: Final[str] = os.getenv('STORAGE_BUCKET_NAME', 'not_set')
And use this env for all buckets that you are planning to actually find.
You can also overwrite this env variable with export when running your own tests. 

We set this env var in setup.cfg under tools pytest.

**Comment by ChameleonTartu** [comment]
@thomasborgen I updated code & tests.

**Review by thomasborgen — APPROVED** [review]
Perfect.

**Pull request merged**